demisto_sdk/commands/content_graph/interface/neo4j/queries/validations.py

```diff
@@ -108,13 +108,17 @@ def validate_toversion(
     }
 
 
-def get_items_using_deprecated(tx: Transaction, file_paths: List[str]):
+def get_items_using_deprecated(
+    tx: Transaction, file_paths: List[str]
+) -> List[Tuple[str, str, List[graph.Node]]]:
```

Comment:
now a `List[DeprecateItemUsage]`, no?

Reply:
Nope, the `DeprecateItemUsage.content_items_using_deprecated` is of type `List[BaseNode]` and not `List[graph.Node]`. The conversion from `graph.Node` to `BaseNode` happens in the `find_items_using_deprecated_items` function.